### ISSUE TITLE
Dance Form Classification

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/DL-Simplified#73]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Dance Form Classification</br>
:red_circle: **Aim** : Create a DL model which will predict the dance forms.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/singhuday/identifythedanceform</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by sristis0202** [comment]
I am interested in contributing to this issue

**Comment by sristis0202** [comment]
Issue No: #73 Dance Form Classification
Full Name: Sristi Sharma
Github Profile Link: https://github.com/sristis0202
Email ID: 2019sristi.sharma@ves.ac.in
Approach: I will use apt DL Algorithms, experiment with GANs to have the best possible accuracy
Participant Role : Contributor(SSOC)

**Comment by 072arushi** [comment]
@sristis0202 you have been assigned the issue. All the best.

**Comment by abhisheks008** [comment]
@sristis0202 updates please.

**Comment by abhisheks008** [comment]
Updates please @sristis0202

**Comment by abhisheks008** [comment]
@sristis0202 updates please.

**Comment by jahenvins** [comment]
Full name : JAHNEVI N S
GitHub Profile Link : https://github.com/jahenvins
Email ID : jahnevi@gmail.com
Approach for this Project : Planning to do pre-processing, EDA and use CNN for classification

**Comment by abhisheks008** [comment]
Issue assigned to you @jahenvins. Try to use more algorithms, check those based on the accuracy scores and find out the best fitted one.

**Comment by abhisheks008** [comment]
Issue has been un-assigned from you due to inactivity.
@jahenvins

**Comment by ayushete02** [comment]
:white_check_mark: 
- Full name : Ayush Shete
- GitHub Profile Link : https://github.com/ayushete02
- Approach for this Project : I will done this by using python 
- What is your participant role? 
Developer

**Comment by stiwari-ds** [comment]
@abhisheks008 Please assign this issue to me.

**Comment by abhisheks008** [comment]
Can you elaborate your approach here, and please follow the template given to you while commenting in an issue.
@stiwari-ds

**Comment by stiwari-ds** [comment]
* Full name : **Siddhant Tiwari**
* GitHub Profile Link : https://github.com/stiwari-ds
* Email ID : siddhant.tiwari.ds@gmail.com
* Participant ID (if applicable): s_tiwari (Devfolio)
* Approach for this Project : I will try the following models/methods for classifying images -
1. Train a CNN from scratch, manually try out a few different configurations
2. Create and train a CNN using KerasTuner to automatically tune for ideal configuration.
3. Transfer learning:
   a. Use VGG and EfficientNet architectures as feature extractors and use embeddings as input to a classifier.
   b. Fine-tune VGG and EfficientNet for the target dataset

* What is your participant role? (Mention the Open Source program): Individual Contributor, SWOC S3 (2023)

**Comment by stiwari-ds** [comment]
> Can you elaborate your approach here, and please follow the template given to you while commenting in an issue. @stiwari-ds

@abhisheks008 Sorry about that. Applied concurrently with another repo and it slipped my mind.

**Comment by abhisheks008** [comment]
Issue assigned to you. If there is any changes requested in the PR made by you, please correct that changes first and then work on this issue.
@stiwari-ds

**Comment by stiwari-ds** [comment]
Thank you. Currently there are no changes requested there, so I will start with this.  
Will keep fixing if an problems arise in the earlier PR.
